src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1378,7 +1378,7 @@ private String init(boolean initFull) {
                 // Set Working Version and Dataset by PersistentID
                 dataset = datasetService.findByGlobalId(persistentId);
                 if (dataset == null) {
-                    logger.warning("No such dataset: "+persistentId);
+                    logger.warning("++++No such dataset: "+persistentId);
```

Comment:
Is this and the one below also an accident? Looks weird.

Reply:
If you want to keep these warning messages in for production, its good to have a bit more description about where the error is happening so its easier to debug from the log files.